### PR TITLE
Accelerate checking k8s-agents and k8s-controllers setup via 1 script

### PR DESCRIPTION
**Pull request by missingcharacter**
```
$ cd /opt/ivy
$ ./k8s-check.sh -e -a -s 'k8s-controllers'
```

### CODE REVIEW

File: roles/kubernetes/files/opt/ivy/k8s-check.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        ;;
+  esac
+done
+
+if [[ -z ${STACK_NAME:-""} ]] ; then
+  usage
+fi
+
+warn 'Always check: /var/log/cloud-init*'
+warn 'You may ran: tail -F /var/log/cloud-init*'
```
Comment (samfreiberg-oh) [inline roles/kubernetes/files/opt/ivy/k8s-check.sh:49]: Should this be?: You may run: ...

**Review by samfreiberg-oh — APPROVED** [review]
Just one minor grammatical question otherwise LGTM.

File: roles/kubernetes/files/opt/ivy/k8s-check.sh (hunk 1 above)
Comment (missingcharacter) [reply roles/kubernetes/files/opt/ivy/k8s-check.sh:49]: Yes, you are right, 1 moment

**Review by lonnixOH — CHANGES_REQUESTED** [review]

**Review by lonnixOH — APPROVED** [review]

**Review by Pleshchuk — APPROVED** [review]

**Review by samfreiberg-oh — APPROVED** [review]
LGTM

**Pull request merged**